### PR TITLE
Adds mysql

### PR DESCRIPTION
**Pull request by gmmeyer**

### CODE REVIEW

**Comment by remh** [comment]
This should live in https://github.com/DataDog/integrations-core

File: mysql/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+## Overview
+
+Get metrics from mysql service in real time to:
+
+* Visualize and monitor mysql states
+* Be notified about mysql failovers and events.
+
+## Installation
+
+Install the `dd-check-mysql` package manually or with your favorite configuration manager
```
Comment (remh) [inline mysql/README.md]: This will come preinstalled with the agent

**Comment by remh** [comment]
Let's add the metrics metadata as well

File: mysql/README.md (hunk 1 above)
Comment (gmmeyer) [inline mysql/README.md]: I will move it there!

**Pull request closed without merging**